Stetskyi_Homework_7/Unit Testing/ImplementedKatas/FizzBuzz/FizzBuzz/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace FizzBuzz
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            for (int i = 1; i < 100; i++)
+            {
+                Console.WriteLine(WriteFizzBuzz(i));
+            }
+        }
+
+        public static string WriteFizzBuzz(int myint)
+        {
+            ExceptionHandler(myint);
```

Comment:
@IhorStetskyi, rename a method, please. (Use verb when write methods)

Reply:
WriteFizzBuzz renamed to IterateThroughArrayAndDisplayResult